packages/carbon-graphs/tests/unit/controls/PairedResult/PairedResultLoad-spec.js

```diff
@@ -367,6 +367,88 @@ describe('Paired Result - Load', () => {
       );
       expect(point.length).toBe(0);
     });
+    it('does not update x axis range if allow calibration is disabled', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.allowCalibration = false;
+      disableCalibrationInput.axis.x.lowerLimit = 500;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(valuesDefault, false, false);
+      disableCalibrationGraph.loadContent(new PairedResult(input));
+
+      expect(disableCalibrationGraph.config.axis.x.domain.upperLimit)
+        .toEqual(disableCalibrationInput.axis.x.upperLimit);
+      expect(disableCalibrationGraph.config.axis.x.domain.lowerLimit)
+        .toEqual(disableCalibrationInput.axis.x.lowerLimit);
+    });
+    it('does not update x axis range if allow calibration is undefined', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.lowerLimit = 500;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(valuesDefault, false, false);
+      disableCalibrationGraph.loadContent(new PairedResult(input));
+
+      expect(disableCalibrationGraph.config.axis.x.domain.upperLimit)
+        .toEqual(disableCalibrationInput.axis.x.upperLimit);
+      expect(disableCalibrationGraph.config.axis.x.domain.lowerLimit)
+        .toEqual(disableCalibrationInput.axis.x.lowerLimit);
+    });
+    it('does not update x axis range if allow calibration is enabled and datapoints are within limits', () => {
+      const disableCalibrationInput = getAxes(axisDefault);
+      disableCalibrationInput.axis.x.allowCalibration = true;
+      disableCalibrationInput.axis.x.lowerLimit = 0;
+      disableCalibrationInput.axis.x.upperLimit = 500;
+      const disableCalibrationGraph = new Graph(disableCalibrationInput);
+      input = getInput(valuesDefault, false, false);
+      disableCalibrationGraph.loadContent(new PairedResult(input));
+
+      expect(disableCalibrationGraph.config.axis.x.domain.upperLimit).toEqual(500);
+      expect(disableCalibrationGraph.config.axis.x.domain.lowerLimit).toEqual(0);
+    });
+    it('updates x axis range if allow calibration is enabled and datapoints are within limits', () => {
```

Comment:
did you mean, `exceed limits`?

Reply:
I would also add another test for `updates x axis range if allow calibration is enabled and datapoints are equal to limits`.

Reply:
[4fa22a2](https://github.com/cerner/terra-graphs/pull/289/commits/4fa22a292797b9c524099f93eea9c283a6c325f2)